Source/Initialization.cpp

```diff
@@ -174,9 +174,12 @@ void InitializeFields(std::array< MultiFab, AMREX_SPACEDIM >&  Mfield,
                 Real z = prob_lo[2] + (k+0.5) * dx[2];
                
                 //x_face 
-                M_xface(i,j,k,0) = (y < 0) ? 1.4e5 : 0.;
-                M_xface(i,j,k,1) = 0._rt;
-                M_xface(i,j,k,2) = (y >= 0) ? 1.4e5 : 0.;
+               //  M_xface(i,j,k,0) = (y < 0) ? 1.4e5 : 0.;
+               //  M_xface(i,j,k,1) = 0._rt;
+               //  M_xface(i,j,k,2) = (y >= 0) ? 1.4e5 : 0.;
+               M_xface(i,j,k,0) = 8.0e5 /sqrt(3.0);
+               M_xface(i,j,k,1) = 8.0e5 /sqrt(3.0);
+               M_xface(i,j,k,2) = 8.0e5 /sqrt(3.0);
```

Comment:
This normalization is needed for the standard problem 4 verification. 
It can be changed into parsing in the future.